app/assets/stylesheets/application.scss

```diff
@@ -27,6 +27,19 @@
 a:hover {
   text-decoration: underline;
 }
+
+footer, nav {
+  span.spacer{
+    margin: 0em 0.5em 0em 0.5em;
+  }
+  a {
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.